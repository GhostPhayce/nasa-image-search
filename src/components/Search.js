import React from "react";
import "../styles/Search.css";

const Search = () => {
    return (
        <>
            <input className="Search__input" type="text"></input>
        </>
    );
};

export default Search;
