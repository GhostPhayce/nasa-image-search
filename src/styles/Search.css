.Search__input {
  max-width: 300px;
  padding: 1rem 1rem;
  font-size: 1.5rem;
  margin: 0 auto;
}
