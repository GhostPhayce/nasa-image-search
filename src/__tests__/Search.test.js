import { render } from '@testing-library/react';
import Search from "../components/Search";

it("renders Search component", () => {
    const { asFragment } = render(<Search />);
    expect(asFragment()).toMatchSnapshot();
});