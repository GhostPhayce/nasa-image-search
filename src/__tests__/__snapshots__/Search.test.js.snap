// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <img
      alt="NASA logo"
      class="App__logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <input
      class="Search__input"
      type="text"
    />
  </div>
</DocumentFragment>
`;

exports[`renders Search component 1`] = `
<DocumentFragment>
  <input
    class="Search__input"
    type="text"
  />
</DocumentFragment>
`;

exports[`renders correctly 1`] = `
<DocumentFragment>
  <input
    class="Search__input"
    type="text"
  />
</DocumentFragment>
`;

exports[`renders yes 1`] = `
<DocumentFragment>
  <input
    class="Search__input"
    type="text"
  />
</DocumentFragment>
`;
